src/sentry/incidents/logic.py

```diff
@@ -949,7 +949,15 @@ def update_alert_rule(
                 "time_window", timedelta(seconds=snuba_query.time_window)
             )
             updated_query_fields.setdefault("event_types", None)
-            updated_query_fields.setdefault("resolution", timedelta(seconds=snuba_query.resolution))
+            if (
```

Comment:
I do this because for dynamic alerts, we've already converted it to a timedelta [here](https://github.com/getsentry/sentry/blob/ceorourke/update-snuba-query-dynamic-rules/src/sentry/incidents/logic.py#L887-L889)